build.sbt

```diff
@@ -23,10 +23,15 @@ libraryDependencies ++= Seq(
   "com.typesafe.akka" %% "akka-actor" % akkaV,
   "commons-codec" % "commons-codec" % "1.10",
   "ch.qos.logback" % "logback-classic" % "1.1.3",
+  "com.typesafe.slick" %% "slick" % "3.0.0",
+  "com.zaxxer" % "HikariCP" % "2.3.3",
```

Comment:
Not a quibble, just curious why this vs c3p0? "It's what I've used in the past" is a completely valid answer :)